src/machine.rs

```diff
@@ -8,6 +8,7 @@ use std::num::NonZeroU64;
 use std::time::Instant;
 
 use log::trace;
+use measureme::{DetachedTiming, EventId, Profiler, StringId};
```

Comment:
Given that people reading this code will likely not be familiar with the measureme API, I think it'd be better to explicitly use `measureme::...` in the code below -- that avoids confusion, in particular for very generic names like `EventId` and `StringId`.